src/equations/bbm_bbm_1d.jl

```diff
@@ -202,9 +208,18 @@ function rhs!(du_ode, u_ode, t, mesh, equations::BBMBBMEquations1D, initial_cond
 
     @timeit timer() "source terms" calc_sources!(dq, q, t, source_terms, equations, solver)
 
-    @timeit timer() "deta elliptic" deta[:]=invImD2 * deta
-    @timeit timer() "dv elliptic" dv[:]=invImD2 * dv
-
+    # To use the in-place version `ldiv!` instead of `\`, we need temporary arrays
+    # since `deta` and `dv` are not stored contiguously
+    @timeit timer() "deta elliptic" begin
+        tmp1[:] = deta
+        ldiv!(tmp3, invImD2, tmp1)
+        deta[:] = tmp3
```

Comment:
I set `tmp1` here and not use `tmp1` instead of `deta` already earlier for the hyperbolic part because of the source terms that need to be computed in between and take `dq`, i.e. we already need to set `deta` and `dv` before.

Reply:
You could get rid of `tmp3, tmp4` and just use `tmp1, tmp2` here, can't you?

Reply:
Yes, you're right. Thanks! Done in 3da50c61524fbd4d9c5f5a89a2c7bd0e724c515d.